documentation/USAGE.rst

```diff
@@ -549,13 +549,61 @@ array. For example, if ``Term01`` is a transient term then ``Term01``
 would appear in the upper left diagonal and the ordering of the
 variable column array would be reversed.
 
-The use of coupled equation is described in detail in
+The use of coupled equations is described in detail in
 :mod:`examples.diffusion.coupled`. Other examples that demonstrate the
 use of coupled equations are :mod:`examples.phase.binaryCoupled`,
 :mod:`examples.phase.polyxtalCoupled` and
 :mod:`examples.cahnHilliard.mesh2DCoupled`. As well as coupling
-equations, true vector equations can now be written in :term:`FiPy`
-(see :mod:`examples.diffusion.coupled` for more details).
+equations, true vector equations can now be written in :term:`FiPy`.
+
+.. attention::
+
+    Coupled equations are not compatible with
+    :ref:`discret-higherOrderDiffusion` terms.  This is not a practical
+    limitation, as any higher order terms can be decomposed into multiple
+    2nd-order equations.  For example, the pair of `coupled Cahn-Hilliard &
+    Allen-Cahn`_ 4th- and 2nd-order equations
+
+    .. math::
```

Comment:
I'm not sure what's going wrong for you, but this one in particular prompted me to discover that the PDF generation was broken in the CI and that my local build environment, in turn, was broken, so that's something.

If you're only seeing the latex, I'm guessing your browser isn't rendering .svg for some reason and your getting the alt-text.

Reply:
This is fixed in local HTML & LaTeX builds as of 0539af8.